api/v1alpha1/condition_consts.go

```diff
@@ -37,3 +37,23 @@ const (
 	// DeletedReason (Severity=Info) documents a condition not in Status=True because the underlying object was deleted.
 	DeletedReason = "Deleted"
 )
+
+// Conditions related to the VirtualMachineImages
+const(
+	// VirtualMachineImageOsTypeSupportedCondition denotes that the OS type in the virtual machine image object is
+	// supported by VMService. A VirtualMachineImageOsTypeSupportedCondition is marked true
+	// - If OS Type is of Linux Family
+	// - If OS Type is supported by hosts in the cluster
+	VirtualMachineImageOsTypeSupportedCondition ConditionType = "VirtualMachineImageOsTypeSupported"
+
+	// VirtualMachineImageSupportedCondition denotes image compatibility with VMService. VMService expects
```

Comment:
Now we have better clarity on path forward, could you rename this to `VirtualMachineImagev1alpha1CompatibleCondition` to be more specific. And the description should state the VirtualMachinieImage will be prepared by VMware specifically for VM Service v1alpha1. (and drop the bit about Photon, for v1alpha1 we will be preparing custom Photon images as well). 

Sorry for renaming churn :(

Reply:
fixed